### ISSUE TITLE
GitFS online documentation is missing a section present in the code

### ISSUE THREAD

**Issue by amendlik** [saltstack/salt#36475]
### Description of Issue/Question
There is a section of documentation in the code that does not appear in the online documentation.
The code is here: 
https://github.com/saltstack/salt/blame/b4a28e2684e91dda7e326efa957507e852c90436/doc/topics/tutorials/gitfs.rst#L781-L786

Using the GitHub RST parser, that section appears:
https://github.com/saltstack/salt/blob/b4a28e2684e91dda7e326efa957507e852c90436/doc/topics/tutorials/gitfs.rst#refreshing-gitfs-upon-push

But it is missing in the SaltStack online documentation:
https://docs.saltstack.com/en/latest/topics/tutorials/gitfs.html#refreshing-gitfs-upon-push

**Comment by gtmanfred** [comment]
Yup, it looks like that is not there in develop either.

Thanks for reporting this.

**Comment by rallytime** [comment]
This is caused by this error here, when building the docs:
```
/root/SaltStack/salt/doc/topics/tutorials/gitfs.rst:918: ERROR: Error in "code-block" directive:
1 argument(s) required, 0 supplied.

.. code-block::

    Cmnd_Alias SALT_GIT_HOOK = /bin/salt-call event.fire_master update salt/fileserver/gitfs/update
    Defaults!SALT_GIT_HOOK !requiretty
    ALL ALL=(root) NOPASSWD: SALT_GIT_HOOK
```
And is fixed by added a `bash` option in the `.. code-block::` line. This is fixed in #36478. 

Thanks for letting us know @amendlik!
